src/components/booking/bookingService.js

```diff
@@ -0,0 +1,60 @@
+import { logger } from '../../middleware/logger.js';
+import { catchAsync } from '../../utils/catchAsync.js';
+import ExpressError from '../../utils/ExpressError.js';
+
+export const bookOne = (Model_1, Model_2) =>
```

Comment:
Instead of passing Model as parameter, import that model inside the service file